Missing objective data for quest 26118 - Seize the Ambassador (Ironforge, Cataclysm Classic)

<!-- READ THIS FIRST

Hello, thanks for taking the time to report a bug!

Before you proceed, please verify that you're running the latest version of Questie. The easiest way to do this is via the Curseforge client, but you can also download the latest version here: https://www.curseforge.com/wow/addons/questie

Questie is one of the most popular Classic WoW addons, with over 30 million downloads. However, like almost all WoW addons, it's built and maintained by a team of volunteers.

If you'd like to help, please consider making a donation. You can do so here: https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=aero1861%40gmail%2ecom&lc=CA&item_name=Questie%20Devs&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted

You can also help as a tester, developer or translator. To do so please join the Questie Discord here https://discord.gg/fYcQfv7

-->


## Description
<!-- Explain in detail what problem you encountered. Is a quest not showing or showing again or an objective not showing or a spawn missing? -->

Hi folks! While running through Dun Morogh quests, I got one from Ironforge Airfield that took me to the Great Seat to talk to the Dark Iron Ambassador. After turning in that quest, we get Seize the Ambassador, which Questie is missing objective data for:

[ERROR] Missing objective data for quest 26118 Arrest Ambassador Slaghammer and bring him to the High Seat

## Wowhead link
<!-- If this issue is about a quest/npc/item/... then please include a link to it from Wowhead: https://wowhead.com/classic or https://wowhead.com/wotlk or https://wowhead.com/cata -->

https://www.wowhead.com/cata/quest=26118/seize-the-ambassador

## Screenshots
<!-- If you can, add a screenshot to help explaining the problem (e.g. of your map where a quest is not shown but the NPC has an ! above him). Simply drag and drop the image in this input field, no need to upload it to any other image platform. -->

Questie Error:
![image](https://github.com/user-attachments/assets/b3762708-5a54-4418-9e42-5451b784e869)

Quest Description:
![image](https://github.com/user-attachments/assets/fefff5e0-8488-46ef-a718-30109350bc1b)

## Questie & WoW version
<!--
Which version of Questie are you using? You can find it by:
- 1. Hovering over the Questie Minimap Icon
- 2. looking at your Questie.toc file (open it with any text editor).
It looks something like this: "v5.9.0" or "## Version: 5.9.0".

Please also state if your quest issue is about Classic Era/SoM or Wotlk.
-->

Questie 10.3.3
WoW 4.4.0 (Cataclysm Classic)

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3675,6 +3675,9 @@ function CataQuestFixes.Load()
         [26116] = { -- Kinelory Strikes
             [questKeys.triggerEnd] = {"Protect Kinelory", {[zoneIDs.ARATHI_HIGHLANDS] = {{54.85,55.55}}}},
         },
+        [26118] = { -- Seize the Ambassador
+            [questKeys.triggerEnd] = {"Arrest Ambassador Slaghammer and bring him to the High Seat", {[zoneIDs.IRONFORGE] = {{39.65,56.85}}}},
+        },
         [26121] = { -- Claim Korthun's End
             [questKeys.preQuestSingle] = {27443},
             [questKeys.exclusiveTo] = {26105},

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3677,7 +3677,7 @@ function CataQuestFixes.Load()
         },
         [26118] = { -- Seize the Ambassador
             [questKeys.triggerEnd] = {"Arrest Ambassador Slaghammer and bring him to the High Seat", {[zoneIDs.IRONFORGE] = {{39.65,56.85}}}},
-            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_EVENT,l10n("Arrest Ambassador Slaghammer"),0,{{"monster",42146}}}},
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_INTERACT,l10n("Arrest Ambassador Slaghammer"),0,{{"monster",42146}}}},
         },
         [26121] = { -- Claim Korthun's End
             [questKeys.preQuestSingle] = {27443},

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3676,7 +3676,8 @@ function CataQuestFixes.Load()
             [questKeys.triggerEnd] = {"Protect Kinelory", {[zoneIDs.ARATHI_HIGHLANDS] = {{54.85,55.55}}}},
         },
         [26118] = { -- Seize the Ambassador
-            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_MOUNT_UP,l10n("Arrest Ambassador Slaghammer and bring him to the High Seat"),0,{{"monster",42146}}}},
+            [questKeys.triggerEnd] = {"Arrest Ambassador Slaghammer and bring him to the High Seat", {[zoneIDs.IRONFORGE] = {{39.65,56.85}}}},
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_EVENT,l10n("Arrest Ambassador Slaghammer"),0,{{"monster",42146}}}},
         },
         [26121] = { -- Claim Korthun's End
             [questKeys.preQuestSingle] = {27443},

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -3676,7 +3676,7 @@ function CataQuestFixes.Load()
             [questKeys.triggerEnd] = {"Protect Kinelory", {[zoneIDs.ARATHI_HIGHLANDS] = {{54.85,55.55}}}},
         },
         [26118] = { -- Seize the Ambassador
-            [questKeys.triggerEnd] = {"Arrest Ambassador Slaghammer and bring him to the High Seat", {[zoneIDs.IRONFORGE] = {{39.65,56.85}}}},
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_MOUNT_UP,l10n("Arrest Ambassador Slaghammer and bring him to the High Seat"),0,{{"monster",42146}}}},
         },
         [26121] = { -- Claim Korthun's End
             [questKeys.preQuestSingle] = {27443},

--- a/Localization/Translations/ExtraObjectives/CataObjectives.lua
+++ b/Localization/Translations/ExtraObjectives/CataObjectives.lua
@@ -2534,6 +2534,18 @@ local cataObjectiveLocales = {
         ["zhTW"] = false,
         ["zhCN"] = false,
     },
+    ["Arrest Ambassador Slaghammer"] = {
+        ["ptBR"] = false,
+        ["ruRU"] = false,
+        ["deDE"] = false,
+        ["koKR"] = false,
+        ["esMX"] = false,
+        ["enUS"] = true,
+        ["frFR"] = false,
+        ["esES"] = false,
+        ["zhTW"] = false,
+        ["zhCN"] = false,
+    },
 }
 
 for k, v in pairs(cataObjectiveLocales) do

